### PR TITLE
291 Update Mogami TS Docs

### PR DESCRIPTION
**Pull request by richardmands**

### CODE REVIEW

File: docs/sdk/typescript.mdx [hunk 1]
```diff
@@ -45,18 +42,25 @@ const keypair = Keypair.fromMnemonicSet(mnemonic)[0]
...
+Airdrops take some time to complete as we make sure the transaction has been confirmed on the blockchain before returning a response. 
 
 ```ts
 const signature = await sdk.requestAirdrop(keypair.publicKey, "42000")
 ```
 
 ### Make a Transfer
+Send Kin from a Keypair (as you need the privateKey) to a publicKey on the blockchain.
+
+You can optionally set the commitment level before returning ('Confirmed', 'Finalized' or Processed), the transaction type, or add a referenceId / referenceType.
```
Comment (RamirezAlex) [inline docs/sdk/typescript.mdx:75]: In the future we might want to include also a tiny of example using the referenceId / referenceType, becase we mention the user can add it, but it is not clear how.

Also I noticed this small indentation issue right below, while you are here, maybe you can fix it. :)
image

**Review by RamirezAlex — APPROVED** [review]
I like it 🎉 , looks great.

I added a small comment for you to consider, though.

File: docs/sdk/typescript.mdx [hunk 2]
```diff
@@ -10,31 +10,28 @@ Learn how to use the Mogami TypeScript SDK
...
-         axios \
-         bip39 \
-         ed25519-hd-key \
-         tslib
+yarn add @mogami/sdk @mogami/keypair @mogami/solana 
 ```
 
 ## Initialize SDK
 
+Initialise the SDK so that you have access to all of Mogami's methods.
```
Comment (beeman) [inline docs/sdk/typescript.mdx:18]: I think here we want to have a note that tells people to configure the `endpoint` property one a self-hosted instance

File: docs/sdk/typescript.mdx [hunk 3]
```diff
@@ -45,23 +42,42 @@ const keypair = Keypair.fromMnemonicSet(mnemonic)[0]
...
+```ts
+makeTransfer({ amount, commitment, destination, owner, referenceId, referenceType, type, }: {
+    amount: string;
+    commitment?: Commitment;
+    destination: string;
+    owner: Keypair;
+    referenceId?: string;
+    referenceType?: string;
+    type?: TransactionType;
+}): Promise<import("../generated").AppTransaction>;
```
Comment (beeman) [inline docs/sdk/typescript.mdx]: I think here we can remove the 'import' part, and return `Promise<AppTransaction>` instead.

**Review by beeman — COMMENTED** [review]
Two small comments, looks great though!

**Pull request merged**